### ISSUE TITLE
Admin Backend Login Support

### ISSUE THREAD

**Issue by LB-Team-igniti** [iMi-digital/magento2-friendly-captcha#27]
Hello to all,
Is it planned to support the backend login in the future? As far as my team and I could tell, this is not the case at the moment. However, from our point of view, it could be a very useful addition. 
Greetings

**Comment by amenk** [comment]
Yes, not supported at the moment, not really planned, but I guess we would accept a PR.
Also consider 2FA to better secure the admin.
